src/kubernetes_backup/kubernetes_backup/backup.py

```diff
@@ -13,23 +14,32 @@
 def backup_to_s3(basedir, export_dir, cluster_label, s3_bucket, s3_prefix):
   ''' back up yamls to S3 '''
   archive_file = os.path.join(basedir, "kubernetes-backup-%s.tar.gz" % cluster_label)
-  print("INFO: Writing local archive file: %s" % archive_file)
+  logging.info(f"Writing local archive file: {archive_file} ...")
```

Comment:
use `logging` module, instead of `print`. and switch to `f""` format which i find easier to read.

Reply:
`f"""` is also the official recommendation for interpolation now in Python.